### PR TITLE
Added a view from which the admin can import the movies

### PR DESCRIPTION
**Pull request by PurtucRobert**
Added the ability to import the movies from a ZIP file containing the a CSV file with the information related to the movies, and one folder with the posters

### CODE REVIEW

File: cinemy/import_movies/apps.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from django.apps import AppConfig
+
+
+class ImportMoviesConfig(AppConfig):
+    default_auto_field = "django.db.models.BigAutoField"
+    name = "import_movies"
+
+    def ready(self):
+
+        from import_movies.signals import handlers
```
Comment (PurtucRobert) [inline cinemy/import_movies/apps.py:10]: Django related import

File: cinemy/import_movies/signals/handlers.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from concurrent.futures import process
```
Comment (PurtucRobert) [inline cinemy/import_movies/signals/handlers.py]: Removed

File: cinemy/import_movies/apps.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (PurtucRobert) [reply cinemy/import_movies/apps.py:10]: Django related import

**Review by maxinasa — APPROVED** [review]
The code looks pretty good.

**Pull request merged**